AB2D-6814: Add patient/ support to CapabilityStatement

colby-seyferth-nava opened the pull request:
## 🎫 Ticket

https://jira.cms.gov/browse/AB2D-6814

## 🛠 Changes

Added the CapabilityStatement.rest.resource element to declare support for the patient export bulk operation.

## ℹ️ Context

The existing CapabilityStatement was human readable, but the expected machine-readable elements were not present for declaring support for the guide/operation

## 🧪 Validation

* Make a request to the v2 metadata element
  * Validate that the CapabilityStatement.rest.resource.type is "Patient"
  * Validate that the CapabilityStatement.rest.resource.operation.name is "export"
  * Validate that the CapabilityStatement.rest.resource.operation.definition is "http://hl7.org/fhir/uv/bulkdata/OperationDefinition/patient-export"

bennavapbc commented:
PR itself looks fine, but looks like the checkstyle plugin failed the build: https://github.com/CMSgov/ab2d/actions/runs/16910140393/job/47909403803?pr=1570

<img width="1083" height="452" alt="image" src="https://github.com/user-attachments/assets/4f802eee-6ab5-40e0-8782-59b898e1da97" />

colby-seyferth-nava commented:
> PR itself looks fine, but looks like the checkstyle plugin failed the build: https://github.com/CMSgov/ab2d/actions/runs/16910140393/job/47909403803?pr=1570
> 
> <img alt="image" width="1083" height="452" src="https://private-user-images.githubusercontent.com/133617495/477095275-4f802eee-6ab5-40e0-8782-59b898e1da97.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTUwMTYzNTksIm5iZiI6MTc1NTAxNjA1OSwicGF0aCI6Ii8xMzM2MTc0OTUvNDc3MDk1Mjc1LTRmODAyZWVlLTZhYjUtNDBlMC04NzgyLTU5Yjg5OGUxZGE5Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgxMlQxNjI3MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04MTM0MDE4ODZiMTkxMTkyMGZmNWUzZTVjZjczZGNmMmY5NGQxNDAyNDUwMjRmYjg3NGZmNDkwNDU5YjczYzY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cncdv9dNM09BDtkQyuytSCP2PrMb4ibAGqIHe7pf6iI">

ope, fixed the style issue, removed the spaces on the empty line. But the integration tests are failing now. Is that related to my change? I am having a hard time following

Sadibhatla reviewed (approved):
Looks great now, thank you for making changes to fix checks
